src/mumble/IPCUtils_linux.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "IPCUtils.h"
+
+#include <unistd.h>
+#include <QtCore/QDir>
+#include <QtCore/QProcessEnvironment>
+#include <QtCore/QString>
+
+namespace Mumble {
+	QDir getRuntimeDir(void) {
+		QString xdgRuntimePath = QProcessEnvironment::systemEnvironment().value(QLatin1String("XDG_RUNTIME_DIR"));
```

Comment:
I don't think we need Qt to read environment variables, do we?

Reply:
it's the [XDG Base Directory standard](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).
Users can change their preferred runtime location by setting the `XDG_RUNTIME_DIR` env var